src/azure-cli/azure/cli/command_modules/vm/aaz/profile_2018_03_01_hybrid/image/_show.py

```diff
@@ -193,7 +193,6 @@ def _build_schema_on_200(cls):
             )
             storage_profile.os_disk = AAZObjectType(
                 serialized_name="osDisk",
-                flags={"required": True},
```

Comment:
Why do we need to remove it?

Reply:
It wasn't supposed to exist. Now just fix it.